test/stat_tracker_test.rb

```diff
@@ -3,38 +3,49 @@
 require './lib/stat_tracker'
 
 class StatTrackerTest < Minitest::Test
+
   def setup
     game_fixture_path = './data/games_fixture.csv'
-    team_fixture_path = './data/teams_fixture.csv'
+    team_fixture_path = './data/teams.csv'
     game_teams_fixture_path = './data/game_teams_fixture.csv'
 
-    fixture_locations = {
+    @fixture_locations = {
     games: game_fixture_path,
     teams: team_fixture_path,
     game_teams: game_teams_fixture_path
     }
 
-    @stat_tracker = StatTracker.from_csv(fixture_locations)
+    @stat_tracker = StatTracker.from_csv(@fixture_locations)
   end
 
   def test_it_exists
     assert_instance_of StatTracker, @stat_tracker
   end
 
-  def test_it_has_attributes
-    assert_equal './data/games_fixture.csv', @stat_tracker.game_path
-    assert_equal './data/teams_fixture.csv', @stat_tracker.team_path
-    assert_equal './data/game_teams_fixture.csv', @stat_tracker.game_teams_path
-  end
+  # def test_it_has_attributes
+  #   assert_equal './data/games_fixture.csv', @stat_tracker.game_path
+  #   assert_equal './data/teams.csv', @stat_tracker.team_path
+  #   assert_equal './data/game_teams_fixture.csv', @stat_tracker.game_teams_path
+  # end
 
-# Game Statistics Tests - Helper Methods #
-  def test_it_can_get_total_goals_per_game 
+  # Game Statistics Tests - Helper Methods #
+  def test_it_can_get_total_goals_per_game
     assert_equal 5, @stat_tracker.total_goals_per_game[2012030221]
     assert_equal 3, @stat_tracker.total_goals_per_game[2012030231]
   end
 
-# Game Statistics Tests - Stat Methods #
+  # Game Statistics Tests - Stat Methods #
   def test_it_can_get_highest_total_score
-    assert_equal 7, @stat_tracker.highest_total_score 
```

Comment:
@sarynm12 Since main now has what is on line 38, will it be overridden when we merge this branch to main? It should be `assert_equal`. I'm still getting used to some of this Github flow so I just wanted to make sure!

Reply:
Another good catch! Sorry about that! I just made those changes and pushed up :)